### ISSUE TITLE
error message concerning missing f16 files in the console

### ISSUE THREAD

**Issue by dirteat** [5H1N0B11/flightgear-mirage2000#199]
Hi there,
with latest snaphot, I get these error messages in the console:

```
4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/glass/glassIR.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/glass/glassIR
    4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage
    4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage
    4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage
    4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage
    4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage
    4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage
    4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage
    4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage
    4.29 [ALRT]:input      can't find "Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage.eff"
    4.29 [ALRT]:input      can't find base effect Aircraft/f16/Models/Effects/fuselage/model-combined-fuselage
    6.41 [INFO]:OSG
    6.41 [WARN]:OSG        Error reading file Fonts/helvetica.txf: file not found
    6.42 [WARN]:OSG        Error reading file Fonts/helvetica.txf: file not found
    6.42 [WARN]:OSG        Error reading file Fonts/helvetica.txf: file not found
    
Thanks,
Cheers,
Chris.

**Comment by dirteat** [comment]
Forgot to say, with 2020.3.5

**Comment by dirteat** [comment]
All right, closed with c10df6773d307f5d5cc944220ec23173afaa2a59
